.github/workflows/desktop-release.yml

```diff
@@ -114,11 +114,17 @@ jobs:
           APPLE_ID_PASSWORD: ${{ secrets.APPLE_ID_PASSWORD }}
           APPLE_TEAM_ID: ${{ secrets.APPLE_TEAM_ID }}
           DAEMON_NAME: ${{ matrix.config.daemon_name }}
-          VITE_APP_VERSION: "${{ needs.get-version.outputs.version }}"
-          # VITE_APP_VERSION: "0.0.100"
+          VITE_VERSION: "${{ needs.get-version.outputs.version }}"
+          # VITE_VERSION: "0.0.100"
           GITHUB_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
-          SENTRY_AUTH_TOKEN: ${{ secrets.SENTRY_AUTH_TOKEN }}
           timeout-minutes: 10
+          VITE_DESKTOP_P2P_PORT: "55000"
+          VITE_DESKTOP_HTTP_PORT: "55001"
+          VITE_DESKTOP_GRPC_PORT: "55002"
+          VITE_DESKTOP_DESKTOP_APPDATA: "Mintter"
```

Comment:
Would this create the data directory in the current folder, or would it still be in `~/Library/Application Support`?

Reply:
it will create the folder in `~/Library/Application Support`

Reply:
How does it know that in dev it needs to create it in the local directory? Is there a special behavior for when it's in dev mode?

Reply:
NVM, I see there's a special behavior in code for when it's production mode. But I'm surprised that the dev data dir will be in your home directory instead of inside the repo. So the dev directory will be `~/appData.local`, and not `./appData.local`, right?

Why is this better than our previous behavior of using `~/Library/Application Support/Mintter.dev`?

Reply:
I also believe that having something called `~/appData.local` is not very descriptive as for something to be in your home directory. It's not clear what app is this.

Reply:
do you think we should go back to store everything inside Application support? is easy to change now! we just need to choose

Reply:
UPDATE: I changed the dev folder location to `~/Library/Application Support/Mintter.dev`. if you want to run another device, you need to change the var `VITE_DESKTOP_APPDATA` to something else